src/aurora/pefile.cpp

```diff
@@ -48,57 +48,86 @@ const Archive::ResourceList &PEFile::getResources() const {
 }
 
 Common::SeekableReadStream *PEFile::getResource(uint32 index, bool UNUSED(tryNoCopy)) const {
-	// Convert from the PE cursor group/cursor format to the standalone
-	// cursor format.
+	ResourceList::const_iterator iter = _resources.begin();
+	if (index >= _resources.size())
+		return 0;
```

Comment:
This should probably even throw.

Compare with what the other archives do in this case: https://github.com/xoreos/xoreos/blob/master/src/aurora/rimfile.cpp#L97 , for example